admin/services/data-access/sql.service.old.js

```diff
@@ -0,0 +1,536 @@
+'use strict'
```

Comment:
@jon-shipley what is the reason for keeping this?
Is it going to be used at all?
I would suggest deleting if not.

Reply:
Good point.  Yes, it will be deleted before I remove the [DNM] flag.

Reply:
Done